pkg/scheduler/uthelper/helper.go

```diff
@@ -154,7 +157,7 @@ func (test *TestCommonStruct) CheckBind(caseIndex int) error {
 	}
 
 	if len(test.Bind) != len(binder.Binds) {
```

Comment:
test.Bind and test.BindsNum are ambiguous, plz seperate them with concret name: )

Reply:
Will refact those names after other pending PRs are merged.

Reply:
Then other prs should also be modified？